http/source_proxy_service.go

```diff
@@ -90,12 +96,17 @@ func (s *SourceProxyQueryService) queryInfluxQL(ctx context.Context, w io.Writer
 	hc := newClient(u.Scheme, s.InsecureSkipVerify)
 	resp, err := hc.Do(hreq)
 	if err != nil {
-		return 0, err
+		return flux.Statistics{}, err
 	}
 	defer resp.Body.Close()
 
 	if err := CheckError(resp); err != nil {
-		return 0, err
+		return flux.Statistics{}, err
 	}
-	return io.Copy(w, resp.Body)
+
+	if _, err = io.Copy(w, resp.Body); err != nil {
+		return flux.Statistics{}, err
+	}
+
+	return flux.Statistics{}, nil
```

Comment:
Why does these not return a complete flux.Statistics struct?

Reply:
This file seems to be talking to the public API, based on the path in the URL.  So there won't be any statistics in the response.

Reply:
I am confused why this implements a query.ProxyQueryService then if it never needs the stats. Perhaps we should just create an issue to investigate as that is not really part of this change.